src/bun.js/bindings/bindings.cpp

```diff
@@ -4048,37 +4072,79 @@ class V8StackTraceIterator {
             return false;
         }
 
-        auto firstColon = line.find(':', openingParenthese + 1);
+        auto lineInner = StringView_slice(line, openingParenthese + 1, closingParenthese);
+        auto colonCount = StringView_count(lineInner, ':', 0);
 
-        // if there is no colon, that means we only have a filename and no line number
+        switch (colonCount) {
+        case 0: {
+            // /path/to/file.js
+            frame.sourceURL = lineInner;
+            break;
+        }
 
-        //   at foo (native)
-        if (firstColon == WTF::notFound) {
-            frame.sourceURL = line.substring(openingParenthese + 1, closingParenthese - openingParenthese - 1);
-        } else {
-            // at foo (/path/to/file.js:
-            frame.sourceURL = line.substring(openingParenthese + 1, firstColon - openingParenthese - 1);
+        case 1: {
+            // /path/to/file.js:
+            // /path/to/file.js:1
+            // node:child_process
+            // C:\Users\dave\bun\file.js
+
+            auto marker1 = 0;
+            auto marker2 = lineInner.find(':', marker1);
```

Comment:
First we get a count which gets the index of every colon (on line 4076)

Then we get the first colon again right here

Do we need to duplicate this work?